src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Color color = Color.red;
+        return color.getValueByInt(new Random().nextInt(5)).toString();
```

Comment:
enum has static method `values()` which returns an array of all its values
let's remove 5 and use this array's length instead, then use random number as index to get the random color from the array